src/crawler.cls.php

```diff
@@ -247,6 +249,72 @@ public static function async_handler($manually_run = false)
 		self::start($manually_run);
 	}
 
+	/**
+	 * Check if crawler can run in the choosen time period
+	 *
+	 * @since 6.1
+	 */
+	public static function _crawler_in_schedule_time()
```

Comment:
Should this be static?

Reply:
function that calls this function is static

![image](https://github.com/user-attachments/assets/acf4e02d-8139-4b1f-a481-ca1b96c657d7)

Reply:
It is different things.

Reply:
Fixed